universal_transfer_operator/src/universal_transfer_operator/integrations/fivetran.py

```diff
@@ -282,7 +320,8 @@ def create_connector(self, fivetran_hook: FivetranHook, group_id: str) -> str:
             # TODO: parse all setup tests status for passed status
         else:
             raise ValueError(api_response)
-        return api_response["data"]["id"]
+        connector_id: str = api_response["data"]["id"]
+        return connector_id
```

Comment:
@utkarsharma2 I noticed a pattern in which, in most functions (sometimes in 2 lines functions), we first assign the return value to a variable and then return it. Could we simplify to return the value without these lines before? That would significantly reduce the number of type hints. I commented in a few places, but it ould be good to do this in all parts of the PR, if possible

Reply:
@tatiana I there are mypy issues with them, therefore we need to explicitly specify the return types,